src/components/structures/MatrixChat.js

```diff
@@ -598,20 +598,32 @@ module.exports = React.createClass({
             theme = 'light';
         }
 
+        // look for the stylesheet elements.
+        // styleElements is a map from style name to HTMLLinkElement.
+        var styleElements = Object.create(null);
         var i, a;
         for (i = 0; (a = document.getElementsByTagName("link")[i]); i++) {
             var href = a.getAttribute("href");
+            // shouldn't we be using the 'title' tag rather than the href?
```

Comment:
https://github.com/vector-im/riot-web/commit/61c5253dbf41fed7deec32d445bfa3c222833a7a fixed that.